msg/mavlink_log.msg

```diff
@@ -1,6 +1,6 @@
 uint64 timestamp		# time since system start (microseconds)
 
-char[50] text
+char[127] text
```

Comment:
fyi the memory increase here is 385 bytes

Reply:
@bkueng so it's better to chunk the uorb messages as well?

Reply:
yeah, but preferably avoiding large strings in Firmware in the first place. That's something for another day though.